app/v2/services/AmendInsurancePoliciesService.scala

```diff
@@ -42,6 +43,7 @@ class AmendInsurancePoliciesService @Inject() (connector: AmendInsurancePolicies
       "INVALID_TAX_YEAR"          -> TaxYearFormatError,
       "INVALID_CORRELATIONID"     -> InternalError,
       "INVALID_PAYLOAD"           -> InternalError,
+      "OUTSIDE_AMENDMENT_WINDOW"  -> RuleOutsideAmendmentWindow,
```

Comment:
Please remove from here and add to the `extraTysErrors` instead so it's clear in the codebase this was introduced for TYS, thanks. This errors map is for Pre TYS i.e. Pre 23-24